src/main/java/com/project/bookstore/model/Book.java

```diff
@@ -0,0 +1,105 @@
+package com.project.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "books")
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = false, unique = true)
+    private String isbn;
+
+    @Column(nullable = false)
+    private BigDecimal price;
+
+    @Column
+    private String description;
+
+    @Column
+    private String coverImage;
+
+    @Column
+    private boolean isDeleted;
+
+    public Long getId() {
```

Comment:
Remove empty @Column. Code will work without them.
```suggestion

   
    private String description;

 
    private String coverImage;

    private boolean isDeleted;

    public Long getId() {
```